kolena/_experimental/dataset/_evaluation.py

```diff
@@ -162,52 +139,28 @@ def _validate_configs(configs: List[TYPE_EVALUATION_CONFIG]) -> None:
                 raise IncorrectUsageError("duplicate eval configs are invalid")
 
 
-def _validate_data(left: pd.DataFrame, right: pd.DataFrame) -> None:
-    if len(left) != len(right):
-        raise IncorrectUsageError("numbers of rows between two dataframe do not match")
-
-
-def _validate_on(left: pd.DataFrame, right: pd.DataFrame, on: TEST_ON_TYPE) -> None:
-    if on is None:
-        raise IncorrectUsageError("on cannot be None")
-    # works for both string and list of string
-    if len(on) == 0:
-        raise IncorrectUsageError("on cannot be empty")
-
-    if isinstance(on, str):
-        if on not in left.columns or on not in right.columns:
-            raise IncorrectUsageError(f"column {on} doesn't exist in target dataframe")
-    elif isinstance(on, list):
-        for col in on:
-            if col not in left.columns or col not in right.columns:
-                raise IncorrectUsageError(f"column {col} doesn't exist in target dataframe")
-
-
-def _align_datapoints_results(
-    df_datapoints: pd.DataFrame,
-    df_result_input: pd.DataFrame,
-    on: TEST_ON_TYPE,
-) -> pd.DataFrame:
-    if not on:
-        return df_result_input
-
-    if isinstance(on, str):
-        on = [on]
-
-    _validate_on(df_datapoints, df_result_input, on)
-    try:
-        df_result = df_datapoints[on].merge(df_result_input, how="left", on=on, validate="one_to_one")
-    except MergeError as e:
-        raise IncorrectUsageError(f"merge key {on} is not unique") from e
-
-    return df_result.drop(columns=on)
+def _validate_datapoint_id_columns(df: pd.DataFrame, datapoint_id_columns: List[str]) -> None:
```

Comment:
can recently abstracted `validate_dataframe_ids` in common.py be reused here?

Reply:
sure